### ISSUE TITLE
3.4.1 switch Line indented incorrectly

### ISSUE THREAD

**Issue by zomew** [squizlabs/PHP_CodeSniffer#2473]
system: windows 10
phpcs --version
PHP_CodeSniffer version 3.4.1 (stable) by Squiz (http://www.squiz.net)

code:
```
<?php
/**
 * Created by PhpStorm.
 * User: Jamers
 * Date: 2019/4/4
 * Time: 9:01
 * File: psr.php
 */

$tmp = '';
switch ($tmp) {
    case '1':
        $ret = 10;
        break;
    default:
        $ret = 0;
        break;
}
```
result:
```
----------------------------------------------------------------------
FOUND 8 ERRORS AND 1 WARNING AFFECTING 4 LINES
----------------------------------------------------------------------
  1 | ERROR   | [x] End of line character is invalid; expected "\n"
    |         |     but found "\r\n"
  8 | WARNING | [ ] PHP version not specified
  8 | ERROR   | [ ] Missing @category tag in file comment
  8 | ERROR   | [ ] Missing @package tag in file comment
  8 | ERROR   | [ ] Missing @author tag in file comment
  8 | ERROR   | [ ] Missing @license tag in file comment
  8 | ERROR   | [ ] Missing @link tag in file comment
 12 | ERROR   | [x] Line indented incorrectly; expected 0 spaces,
    |         |     found 4
 15 | ERROR   | [x] Line indented incorrectly; expected 0 spaces,
    |         |     found 4
----------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 144ms; Memory: 6MB

```

```
system: FREEBSD 11.2
# php phpcs.phar --version
PHP_CodeSniffer version 3.4.1 (stable) by Squiz (http://www.squiz.net)

result:
--------------------------------------------------------------------------
FOUND 7 ERRORS AND 1 WARNING AFFECTING 3 LINES
--------------------------------------------------------------------------
  8 | WARNING | [ ] PHP version not specified
  8 | ERROR   | [ ] Missing @category tag in file comment
  8 | ERROR   | [ ] Missing @package tag in file comment
  8 | ERROR   | [ ] Missing @author tag in file comment
  8 | ERROR   | [ ] Missing @license tag in file comment
  8 | ERROR   | [ ] Missing @link tag in file comment
 12 | ERROR   | [x] Line indented incorrectly; expected 0 spaces, found 4
 15 | ERROR   | [x] Line indented incorrectly; expected 0 spaces, found 4
--------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------

Time: 59ms; Memory: 4MB
```

**Comment by gsherwood** [comment]
If you don't specify a coding standard, the PEAR coding standard is used by default. The PEAR coding standard wants CASE statements indented to the same level as the SWITCH statement, like this:
```php
$tmp = '';
switch ($tmp) {
case '1':
    $ret = 10;
    break;
default:
    $ret = 0;
    break;
}
```

If you don't want that type of indenting, I suggest using PSR2, although I think every other standard also asks for the CASE statements to be indented one level.

**Comment by zomew** [comment]
ok,i'm got it. thanks very much.

```
php phpcs.phar psr.php --standard=PSR2
```
